packages/react-native/scripts/react_native_pods.rb

```diff
@@ -371,6 +371,17 @@ def rct_cxx_language_standard()
   return Helpers::Constants.cxx_language_standard
 end
 
+def print_jsc_removal_message()
+  puts ''
+  puts '=============== JavaScriptCore is being moved ================'.yellow
```

Comment:
nit
```suggestion
  puts '=============== JavaScriptCore is being moved ==============='.yellow
```
In the image in the summary, the first line is longer than the last line by 1 char